src/jmclient/support.py

```diff
@@ -247,19 +249,28 @@ def check_max_fee(fee):
     return check_max_fee
 
 
-def choose_orders(offers, cj_amount, n, chooseOrdersBy, ignored_makers=None,
-                  pick=False, allowed_types=["sw0reloffer", "sw0absoffer"],
-                  max_cj_fee=(1, float('inf'))):
+def choose_orders(offers: List[dict],
+                  cj_amount,
+                  num_counterparties: int,
+                  chooseOrdersBy,
+                  ignored_makers: Optional[List[str]] = None,
+                  pick: bool = False,
+                  allowed_types: List[str] = ["sw0reloffer", "sw0absoffer"],
+                  max_cj_fee: Tuple = (1, float('inf'))) -> Tuple[Optional[dict], int]:
     is_within_max_limits = _get_is_within_max_limits(
         max_cj_fee[0], max_cj_fee[1], cj_amount)
     if ignored_makers is None:
         ignored_makers = []
+    fee_per_kb = jm_single().bc_interface.estimate_fee_per_kb(
+        jm_single().config.getint("POLICY", "tx_fees"), randomize=False)
     #Filter ignored makers and inappropriate amounts
     orders = [o for o in offers if o['counterparty'] not in ignored_makers]
     orders = [o for o in orders if o['minsize'] < cj_amount]
     orders = [o for o in orders if o['maxsize'] > cj_amount]
     #Filter those not using wished-for offertypes
     orders = [o for o in orders if o["ordertype"] in allowed_types]
+    #Filter those not accepting our tx feerate
```

Comment:
Yes, agreed, this is what I suggested in [my comment](https://github.com/JoinMarket-Org/joinmarket-clientserver/issues/721#issuecomment-1824627009) too.